lib/saml_idp_constants.rb

```diff
@@ -14,6 +14,7 @@ module Constants
       IAL2_STRICT_AUTHN_CONTEXT_CLASSREF = "#{IAL_AUTHN_CONTEXT_PREFIX}/2?strict=true".freeze
       IALMAX_AUTHN_CONTEXT_CLASSREF = "#{IAL_AUTHN_CONTEXT_PREFIX}/0".freeze
 
+      PASSWORD_AUTHN_CONTEXT_CLASSREF = 'urn:oasis:names:tc:SAML:2.0:ac:classes:PasswordProtectedTransport'.freeze
```

Comment:
For Monday, consider looking at including other "lower strength" AuthnContexts from the [spec](https://docs.oasis-open.org/security/saml/v2.0/saml-authn-context-2.0-os.pdf) and having an array that can be compared against.

Reply:
Added regular password (not protected transport) too for now